src/AffectedUserFlow/CodeInput/CodeInputForm.tsx

```diff
@@ -52,7 +52,11 @@ const CodeInputForm: FunctionComponent = () => {
 
   const handleOnChangeText = (code: string) => {
     setErrorMessage("")
-    setCode(code)
+    if (code && !code.match(/^\d+$/)) {
```

Comment:
Thoughts on naming `!code.match(/^\d+$/)`?

```
const codeContainsNonDigitChars = !code.match(/^\d+$/)
if (code && codeContainsNonDigitChars) {
...
```

Reply:
might need to be something more like:
```typescript
const codeIsValid = code && !code.match(/^\d+$/)
```
as `!code.match(/^\d+$/)` would error if code is null

Reply:
@johnschoeman This check is there already - in the if statement.

@devinjameson Since the naming in this file uses code as local state as well as an argument for this function, I renamed code in this instance. Also, the codeContainsNonDigitChars has to be a function because of this. Let me know what you think of the changes.